### DISCUSSION TITLE
Any support for charts?

### DISCUSSION THREAD

**Discussion by anderspitman** [charmbracelet/bubbletea#999] [Q&A] [answered]
I'm working on a chart-heavy TUI. Currently experimenting with [termui](https://github.com/gizak/termui), but it doesn't seem actively maintained. I also need solid mouse support which bubbletea + bubblezone appear to provide. Has anyone implemented a charting library for bubbletea? My search came up empty.

**Comment by meowgorithm** [comment, marked as answer]
Hi! Have a look at [ntcharts](https://github.com/NimbleMarkets/ntcharts).

**Reply by anderspitman** [reply]
Those look great! Even support dates.
